dotcom-rendering/src/web/experiments/tests/sign-in-gate-mandatory-long-testrun.ts

```diff
@@ -0,0 +1,189 @@
+import type { ABTest } from '@guardian/ab-core';
+import { getCountryCodeSync } from '../../lib/getCountryCode';
+import { setOrUseParticipations } from '../lib/ab-exclusions';
+
+// Flag to determine whether the canRun function 'setOrUseParticipations' will set a participation (true)
+// or use localstorage participation key to decide canRun result (false)
+const setParticipationsFlag = true;
+
+export const signInGateMandatoryLongTestRunUk: ABTest = {
+	id: 'SignInGateMandatoryLongTestRunUk',
+	start: '2022-09-20',
+	expiry: '2022-10-01',
+	author: 'vlbee',
+	description:
+		'Test run for long mandatory test - Show sign in gate to global users on 3rd article view of simple article templates, with higher priority over banners and epic.',
```

Comment:
For data that is identical over many tests, you could create a base that you spread in actual tests:

```
const sharedTestData = {
	start: '2022-09-20',
	expiry: '2022-10-01',
	author: 'vlbee',
	description:
		'Test run for long mandatory test - Show sign in gate to global users on 3rd article view of simple article templates, with higher priority over banners and epic.',
} as const;

// later…

export const signInGateMandatoryLongTestRunUk: ABTest = {
	...sharedTestData,
}
```